packages/block-editor/src/components/date-format-picker/index.js

```diff
@@ -129,6 +129,7 @@ function NonDefaultControls( { format, onChange } ) {
 	return (
 		<VStack>
 			<CustomSelectControl
+				__next40pxDefaultSize
```

Comment:
Seen in the Date block.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/bb29c35b-afa0-4f93-9d0e-2d21640413fc" alt="DateFormatPicker, before" width="275">|<img src="https://github.com/user-attachments/assets/d9ddced6-6ad4-4878-8339-325c8f9792f1" alt="DateFormatPicker, after" width="275">|